packages/playground/src/domains/prompt-blocks/hooks/__tests__/use-stored-prompt-blocks.editor.msw.test.tsx

```
@@ -0,0 +1,109 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { http, HttpResponse } from 'msw';
+import { describe, expect, it } from 'vitest';
+
+import { server } from '@/test/msw-server';
+import { makeWrapper, TEST_BASE_URL, waitForMutationsIdle } from '@/test/render';
+import {
+  useStoredPromptBlock,
+  useStoredPromptBlockMutations,
+  useStoredPromptBlocks,
+} from '../use-stored-prompt-blocks';
```

ESLint

[error] `../use-stored-prompt-blocks` import should occur before import of `@/test/msw-server`

(import/order)